### PR TITLE
Add community Resource. Ghost CMS Docs page.

### PR DESCRIPTION
**Pull request by Adammatthiesen**
Add new community resource for Ghost CMS as well as a reference to the community resource in the section containing information about the [slug].astro page.  Thank you astro for such a great baseline of a platform!

#### Description (required)

Add new community resource for Ghost CMS as well as a reference to the community resource in the section containing information about making the [slug].astro page work in SSR mode. 

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by at-the-vr — APPROVED** [review]
LGTM, I have only 1 question as a comment

File: src/content/docs/en/guides/cms/ghost.mdx [hunk 1]
```diff
@@ -189,7 +189,7 @@ const posts = await ghostClient.posts
 
 ### Generating pages
 
-The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post.
+The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post in Static Generation Mode, for a SSR reference check [Community Resources](#community-resources) entry by Adam.
```
[the remark this answers is not in the document]
Comment (Adammatthiesen) [reply src/content/docs/en/guides/cms/ghost.mdx:192]: for the Original example [slug] works only in SSG mode, my example of [slug] only works in SSR mode, the slug page was the only page I had to significantly modify between the two modes.

[the remark this answers is not in the document]
Comment (at-the-vr) [reply src/content/docs/en/guides/cms/ghost.mdx:192]: I meant like this (or the SSG) just trying to be accurate 
```suggestion
The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post in Static Site Generation Mode, for a SSR reference check [Community Resources](#community-resources) entry by Adam.
```

**Comment by Adammatthiesen** [comment]
Fixed as per @at-the-vr 's comment. about Static 'site' generator

File: src/content/docs/en/guides/cms/ghost.mdx [hunk 2]
```diff
@@ -189,7 +189,7 @@ const posts = await ghostClient.posts
 
 ### Generating pages
 
-The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post.
+The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post in Static Site Generation Mode, for a SSR reference check [Community Resources](#community-resources) entry by Adam.
```
Comment (sarah11918) [inline src/content/docs/en/guides/cms/ghost.mdx:192]:
```suggestion
The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post.
```
No need to update our existing content! The link at the bottom is fine.

File: src/content/docs/en/guides/cms/ghost.mdx [hunk 3]
```diff
@@ -286,5 +286,6 @@ To deploy your site visit our [deployment guide](/en/guides/deploy/) and follow
 
 - You can get started with this [Astro Ghost Starter](https://github.com/PhilDL/astro-starter-ghost).
 - Andre Carrera wrote [a blog post](https://andr.ec/posts/astro-ghost-blog/) showing how to use Ghost's SDK to embed posts and previews in your Astro project.
+- Adam Matthiesen wrote [a blog post](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) showing how to deal with SSR changes with Ghost, as well as better layout of the Ghost CONTENT_API.
```
Comment (sarah11918) [inline src/content/docs/en/guides/cms/ghost.mdx:289]:
```suggestion
- [Astro + Ghost + Tailwind CSS](https://andr.ec/posts/astro-ghost-blog/) by Andre Carrera
- [Ghost CMS & Astro](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) by Adam Matthiesen
```
I updated these to match our more current format, which uses the blog title as better linked text.

**Review by sarah11918 — APPROVED** [review]
Thank you for adding your helpful article here, @Adammatthiesen , and welcome to Team Docs! 🥳

**Pull request merged**